### ISSUE TITLE
Incompatibilty with Unicode? Illustration: "·"

### ISSUE THREAD

**Issue by Frenzie** [XCMer/fuzzy-window-switcher#5]
If you have this very bug report open in your browser, you should be getting

```
Traceback (most recent call last):
  File "./fuzzy-windows", line 196, in hotkey
    self.toggle()
  File "./fuzzy-windows", line 178, in toggle
    self.windowList.refresh()
  File "./fuzzy-windows", line 53, in refresh
    if name in self.ignored_windows:
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 10: ordinal not in range(128)

```

**Comment by XCMer** [comment]
I tried this but it didn't trigger an error for me. However, I'll dig deeper into this.

**Comment by Frenzie** [comment]
In that case it might be an issue indirectly caused by http://packages.debian.org/wheezy-backports/python-configparser

However, without that package, fuzzy-windows says ConfigParser doesn't exist. Yet the description there says it should be in Python 2.x by default. Strange.

**Comment by Frenzie** [comment]
Apparently this problem was fixed by restarting. Must've been some kind of error between keyboard and monitor, although I'm not quite sure what I should've done differently. It's not like there's an entry for Python in `/etc/init.d` or anything.

**Comment by XCMer** [comment]
I'll mark this issue as closed for now.

**Comment by Frenzie** [comment]
Btw, I just realized I wasn't sufficiently clear. The problem that was actually fixed is that Python 2.7's ConfigParser started working after restarting, but I don't understand what changed. The problem persists with the lowercase configparser backported from Python 3.2, which I since deleted, restarted, and everything's still working fine. But I didn't actually install any extra packages or anything, so I don't get why yesterday `import ConfigParser` would say file not found and now it is. 

In any case, as I already said it'd seem to be a problem with me or Python—not with fuzzy-windows.
